src/styles/_common/_colors.scss

```diff
@@ -80,6 +80,12 @@ $palette: (
     'error': #ff6b6b,
     'info': #00adef,
     'text': #fff
+  ),
+  'footer': (
+    'background': #fafafa,
```

Comment:
Background name is very subjective, it could be background-image, background-position, etc.
Be more specific.